subsys/greybus/gpio.c

```diff
@@ -66,10 +66,13 @@ static uint8_t gb_gpio_line_count(struct gb_operation *operation)
 {
 	struct gb_gpio_line_count_response *response;
 	const struct device *dev;
+	struct gb_bundle *bundle = gb_operation_get_bundle(operation);
+	__ASSERT_NO_MSG(bundle != NULL);
+	unsigned int cport_idx = operation->cport - bundle->cport_start;
 	const struct gpio_driver_config *cfg;
 	uint8_t count;
 
-	dev = gb_cport_to_device(operation->cport);
```

Comment:
would prefer to continue to use `gb_cport_to_device()`

Reply:
Sure, will revert the change, can we use `dev = gb_cport_to_device(operation->cport)` in spi,i2c as well, then we could achieve multiple cports per bundle with very few changes